R/check_package.R

```diff
@@ -1,4 +1,22 @@
-#' A standardised test for R packages
+#' Run the complete set of standardised tests on a package
+#'
+#' A convenience function that runs all packages related tests in sequence.
+#' The details lists the relevant functions.
```

Comment:
Alternatively to the suggestion, you can leave it as it, but as 'details' is plural, then 'list' has to be without 's'.
```suggestion
#' The details section lists the relevant functions.
```